pkg/cimfs/cim_writer_windows.go

```diff
@@ -62,6 +62,33 @@ func Create(imagePath string, oldFSName string, newFSName string) (_ *CimFsWrite
 	return &CimFsWriter{handle: handle, name: filepath.Join(imagePath, fsName)}, nil
 }
 
+// CreateMergedCim creates a new merged CIM from the CIMs provided in the `cimPaths` array.  CIM at index 0 is
+// considered to be the topmost CIM and the CIM at index `cimPath.length-1` is considered the base CIM
```

Comment:
I think it would be more untuitive to make `cimPaths[0]` the base CIM. Is there a reason we don't?

Reply:
It's just how CimFS expects the layers to be added. The CimAddFstoMergedImage call expects that the topmost layer is added first and bottom most layer is added last. But I have updated the function to take cim paths in base to topmost order.